tests/tests/test_guest_measurement.py

```diff
@@ -28,13 +28,58 @@ def test_guest_measurement_check_rtmr(qm):
     """
     Boot measurements check
     """
-    qm.run()
+    with Qemu.QemuMachine() as qm:
+        qm.run()
 
-    m = Qemu.QemuSSH(qm)
+        m = Qemu.QemuSSH(qm)
 
-    deploy_and_setup(m)
+        deploy_and_setup(m)
 
-    m.check_exec('tdrtmrcheck')
+        m.check_exec('tdrtmrcheck')
 
-    qm.stop()
+        qm.stop()
 
+def test_guest_measurement_trust_authority_success():
```

Comment:
i m wondering if we have to put these tests in a separate file, this file is initially reserved for tests of measurement values. i would suggest the new file name `test_guest_attestation_ita` or `test_guest_attestation_quote`